template/.github/workflows/build.yml.j2

```diff
@@ -0,0 +1,308 @@
+---
+name: Stackable Build Pipeline
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - trying
+      - "renovate/**"
+    tags:
+      - "*"
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_INCREMENTAL: '0'
+  CARGO_PROFILE_DEV_DEBUG: '0'
+  RUSTFLAGS: "-D warnings"
+  RUSTDOCFLAGS: "-D warnings"
+  RUST_LOG: "info"
+  PRODUCT_NAME: {[ operator.product_string }]
+  DEV_REPO_HELM_URL: https://repo.stackable.tech/repository/helm-dev
+  TEST_REPO_HELM_URL: https://repo.stackable.tech/repository/helm-test
+  STABLE_REPO_HELM_URL: https://repo.stackable.tech/repository/helm-stable
+
+jobs:
+  # Identify unused dependencies
+  run_udeps:
+    name: Run Cargo Udeps
+    runs-on: ubuntu-latest
+    env:
+      RUSTC_BOOTSTRAP: 1
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions-rs/toolchain@v1.0.7
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: Swatinem/rust-cache@v1.3.0
+        with:
+          key: udeps
```

Comment:
Do these keys need to be specific to the repos? If so we should include the name of the operator maybe?

Reply:
no, they don't

You can remove the cache for now. It won't be used: https://github.com/Swatinem/rust-cache/issues/9#issuecomment-1030048800

Reply:
Ignore me, sorry. This was my mistake. Cache will be used for udeps, it won't be used for cargo-deny.

Reply:
discussed directly with Lars:
Cache is not enabled for cargo-deny, which this comment refers to. For udeps it is used and enabled.